lib/modules/datasource/npm/get.spec.ts

```diff
@@ -563,69 +565,101 @@ describe('modules/datasource/npm/get', () => {
     `);
   });
 
-  it('discards cache with no revision', async () => {
-    setNpmrc('registry=https://test.org\n_authToken=XXX');
-
-    packageCache.get.mockResolvedValueOnce({
-      some: 'result',
-      cacheData: { softExpireAt: '2099' },
-    });
+  describe('cache', () => {
+    const mockTime = (time: string) => {
+      const value = DateTime.fromISO(time).valueOf();
+      Settings.now = () => value;
```

Comment:
use vitest time mocks. it will still work when we drop luxon. In near future the buildin date api will be expanded so we don't need libs like luxon.